packages/barcode-scanning/ios/Plugin/BarcodeScannerHelper.swift

```diff
@@ -200,4 +227,114 @@ public class BarcodeScannerHelper {
     private static func convertDataToJsonArray(_ data: Data) -> [UInt8] {
         return [UInt8](data)
     }
+
+    private static func extractAddressProperties(_ address: BarcodeAddress) -> JSObject {
+        var result = JSObject()
+        result["addressLines"] = address.addressLines
+        result["type"] = address.type.rawValue
+        return result
+    }
+
+    private static func extractCalendarEventProperties(_ calendarEvent: BarcodeCalendarEvent) -> JSObject {
+        var result = JSObject()
+        result["description"] = calendarEvent.eventDescription
+        result["end"] = calendarEvent.end
```

Comment:
Shouldn't a JSObject of type `CalendarDateTime` be created here? You can also just return a timestamp or something similar (if that's possible), but it should behave the same on Android and iOS. Let me know if you need help. 🙌

Reply:
I would actually just prefer an ISO 8601 string, but I don't know if that's possible.

Reply:
I think I have found a good solution. I merge this and then push my changes directly to main/next.